### DISCUSSION TITLE
📣 Call for participants in a GitHub Copilot research study

### DISCUSSION THREAD

**Discussion by ryanjsalva** [community/community#37928] [Copilot Conversations]
Hello, friends! 👋

The GitHub Copilot team needs your help shaping the future of AI-assisted development. We're scheduling one-hour interviews with developers November 7-18. No preparation is necessary, and you'll be compensated with a $125 gratuity. Before scheduling an interview, we have just a few questions. Please  [fill out our quick survey here](https://github.surveymonkey.com/r/generativeAI). We’ll reach out if you’re a fit for the study. Thank you!

**Comment by JOHAN10p** [comment]
wow intesresting

**Comment by cmp-nct** [comment]
I find it irritating how few people are writing here in general, the "feedback" board is almost dead.
Is it that Copilot is so rarely adopted by developers at this point or do the people just not find their way into this board ?

My advise for your team is to listen to the many complaints you find here from the past half year, most people complain about the same things.
Either crashing/hanging plugins or bad behavior when completing. Both needs a fix

**Reply by Regenhardt** [reply]
I for one just didn't find my way here until I looked really hard. Until now, I had no reason to come here.
